src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Converters/ItemTappedEventArgsConverter_Tests.cs

```diff
@@ -16,7 +16,7 @@ public class ItemTappedEventArgsConverter_Tests
             // We know it's deprecated, still good to test it
 #pragma warning disable CS0618 // Type or member is obsolete
                 new object[] { new ItemTappedEventArgs(null, expectedValue), expectedValue },
```

Comment:
@jfversluis inconsistent indents after cleanup

Reply:
Thanks! Interesting, because of the region?